testdata/storage/csi/csi-hostpath-plugin.yaml

```diff
@@ -34,6 +34,16 @@ spec:
       labels:
         app: csi-hostpathplugin
     spec:
+      affinity:
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - csi-hostpathplugin
+            topologyKey: kubernetes.io/hostname
```

Comment:
If the project has nodeselectors, we don't need to update this manifest.

Reply:
@qinpingli  @chao007  Is the concern addressed here ?

Reply:
I think it's unnecessary, but from @chao007 test, looks like this will not affect the csi host plugin pod schedule.
Anyway, I suggest we can revert this update.

Reply:
@chao007  Could you revert the change as @qinpingli's comments above.